File: src/main/java/it/contrader/dto/PlayerDTO.java
package it.contrader.dto;

import it.contrader.model.Team;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor



public class PlayerDTO {
	
	private long id;
	
	private String player_name;
	private String player_surname;
	private int age;
	private int actualMarketValue;
	private int previousMarketValue;
	private String position;
	
	private Team team;
//	private int team_id;
	
	
}

File: src/dto/playerdto.ts
import { TeamDTO } from './teamdto';



export class PlayerDTO {

   	id:number;
    team:TeamDTO;
	player_name:string;
	player_surname:string;
	age:number;
	actualMarketValue:number;
	previousMarketValue:number;
	position:string;
	//team_id:number;
	constructor(){};
}